plugin/build.gradle

```diff
@@ -9,6 +9,44 @@ ext {
 
 apply from: "../scripts/publish-module.gradle"
 
+def khronosOpenxrTempDir = "${buildDir}/temp/khronos_openxr_loader_aar"
+
+task extractKhronosOpenxrLoader {
+    def khronosOpenxrLoader = configurations.detachedConfiguration(dependencies.create("org.khronos.openxr:openxr_loader_for_android:$versions.openxrVersion"))
+
+    doLast {
+        copy {
+            from zipTree(khronosOpenxrLoader.singleFile)
+            into khronosOpenxrTempDir
+        }
+    }
+}
+
+task copyKhronosOpenxrLoader(dependsOn: extractKhronosOpenxrLoader) {
+    def khronosOpenxrTargetDirArm64 = file("../thirdparty/khronos_openxr_sdk/arm64-v8a/arm64-v8a")
+    def khronosOpenxrTargetDirX86_64 = file("../thirdparty/khronos_openxr_sdk/x86_64/x86_64")
```

Comment:
Should those two be temp dirs as well since we're deleting them on cleanup?

Reply:
That makes sense! Changed in my latest push